foundation/foundation-mda/src/main/resources/templates/cucumber.test.cdi.context.java.vm

```diff
@@ -15,7 +15,7 @@ import io.smallrye.reactive.messaging.memory.InMemoryConnector;
  *
  * Originally generated from: ${templateName} 
  */
-public class TestCdiContext extends PipelinesCdiContext {
+public class TestCdiContext implements CdiContext {
```

Comment:
Q: Do we not need the `PipelineMessagingConfig` ?

Reply:
It will be included automatically by virtue of the typical runtime CDI configuration.  This CDI context is only for adding test-specific on top of the runtime configuration.